src/components/summary/_macro.njk

```diff
@@ -9,24 +9,27 @@
     {% if params.hub %}
         {% set className = className + " ons-summary--hub" %}
     {% endif %}
+
+
     <div class="{{ className }}">
         {% for summary in params.summaries %}
             {% if summary.summaryTitle %}
                 <h2 class="ons-summary__title ons-u-mb-m{{ " ons-u-mt-m" if loop.index > 1 else "" }}">{{ summary.summaryTitle }}</h2>
                 {% set titleSize = "3" %}
             {% endif %}
             {% for group in summary.groups %}
-                <div {% if group.id %}id="{{ group.id }}" {% endif %} class="ons-summary__group">
+                <div {% if group.id %}id="{{ group.id }}" {% endif %} class="{{"ons-summary__group--card" if params.variant == 'card' and not params.hub else "ons-summary__group"}}">
```

Comment:
This logic can be simplified and repetition removed if you just set a var at the beginning and then add it to the end of these classes. Adding nothing if the condition isn't met